### ISSUE TITLE
ember/new-module-imports warning about `notifyPropertyChange`

### ISSUE THREAD

**Issue by alexlafroscia** [ember-cli/eslint-plugin-ember#388]
It seems like the `new-module-imports` rule is warning about using `Ember.notifyPropertyChange`. It suggests importing like this instead:

```js
import { notifyPropertyChange } from '@ember/object';
```

That isn't actually a valid import, though. It's not listed in the [API docs](https://emberjs.com/api/ember/release/modules/@ember%2Fobject), and the build fails if you attempt to actually import from there.

**Comment by rwjblue** [comment]
`import { notifyPropertyChange } from '@ember/object';` is correct.

Not sure where things are going wrong with the API docs, but it is properly documented as coming from `@ember/object`:

https://github.com/emberjs/ember.js/blob/c17949ffae310734a2254695051cb37ff055768a/packages/%40ember/-internals/metal/lib/property_events.ts#L29-L30

> the build fails if you attempt to actually import from there.

Update your version of ember-cli-babel -> babel-plugin-ember-modules-api-polyfill -> ember-rfc176-data (most of the time you'd just bump the min version of ember-cli-babel).

**Comment by rwjblue** [comment]
FWIW, it was added in ember-rfc176-data@0.3.6, and both ember-cli-babel@6.18.0 and ember-cli-babel@7.2.0 use that version as a minimum version. However, you _can_ get the ember-rfc176-data update all the way back to ember-cli-babel@6.8.2...
